Fix RubyConf Austria 2026 information

misabegovic opened the pull request:
Invalid date and website is still not ready (have to remove the website builder stuff generated by godaddy).
Kinda wasn't ready for adding the conference to this list yet, it is still a few months away, but we can keep this info.

Thanks!

marcoroth reviewed (approved):
Hey @misabegovic, thanks for the PR!

Sorry if I added this prematurely, just wanted to get it on for visibility, but happy to hold it back for now if you prefer!

misabegovic commented:
> Hey @misabegovic, thanks for the PR!
> 
> Sorry if I added this prematurely, just wanted to get it on for visibility, but happy to hold it back for now if you prefer!

Hey Marco.

Thank you for doing that. I definitely value your support and I am grateful for you adding the conference here.

Just wish I had removed the website builder stuff generated by godaddy sooner on my end. Some folks apparently used the website already to subscribe to the (invalid) newsletter, etc... That's how I found out it's added to the list btw, a speaker expressing interest to join since he thought the conference is this year or in a few months 😅

Love your contributions and your support man.
Please don't stop doing what you do.

Here's a brain dump with information I have for now (will be on the website later)(please excuse the thoughts in the brackets):

The actual date is confirmed (🤞). 
I booked Das MuTh theater in Vienna on 30.05.2026., this is a musical theater located in the center of the city and right next to a beautiful park. Fully accessible for everyone, supports up to 400 participants, it is a beautiful theatre with wonderful acoustics (and more importantly a full production setup that is hard to mess up, with their own streaming & recording services etc etc).

On the evening of 30.05.2026. an after event will be planned (location tbd... thinking about Sperling, the cafe/restaurant in Augarten, the large park right next to the venue which is rather nice, and is a place where renowned musicians performed in the past - Mozart, Beethoven and Schubert ). 

Depending on how strong my intrusive thoughts about hosting an evening concert at the venue become, the after party might take place at the venue as well, or we might have a slightly longer day that ends with some sort of concert. (I am already kind of, sort of, semi seriously looking for a concert pianist ... I already know who the concert pianist might be, I just haven't asked them yet... come on, musical theater, Vienna, and we have a grand concert piano as part of the venue that is booked... I think this thought might have become too strong to be rejected by now). I do think I want to merge classical music with this conference, I think it fits the conference, just not completely sure how it will fit the final program.

Musical performances by speakers are also welcomed (playing of instruments... not sure about singing though), if someone wants to (as one of the confirmed speakers already does, which I think is amazing).

We will have a Vienna.rb meetup on 29.05.2026., organized in collaboration with lovely Carmen and Hans.

On Sunday an optional hike, hangout, or something similar will be planned as well.

It is expected to be a single day, single track for the actual conference. However, once I get a better feeling for the space this might change, since I believe we will have more rooms available. (mind you, EuRuKo 2024 started out smaller as well... but this time I actually want to put more focus on creating a more meaningful experience for folks participating, so I might just end up adding a few more workshops... not sure if it is going to become a second track - tbd).

These are the current key values I am working with:
- long lasting impact
- free flow of artistry and creativity
- passion
- human connection

Together with Zehra (the EuRuKo designer and mc of 2nd track) I am working on preparing the visual identity of the conference and we should have the first version of the website up and running in a few months.

2 high profile speakers are already booked (will be revealed in a few months) and a CFP will be announced later this year once I have a much better idea of what the full program might look like.

End of brain dump.
